### PR TITLE
Add .dockerignore file

### PR DESCRIPTION
**Pull request by luigibk**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: vars/sectionBuildAndTest.groovy [hunk 1]
```diff
@@ -101,6 +101,7 @@ def call(params) {
 
             pcr.callAround('dockerbuild') {
               timeoutWithMsg(time: 30, unit: 'MINUTES', action: 'Docker build') {
+                writeFile file: '.dockerignore', text: libraryResource('uk/gov/hmcts/.dockerignore_build')
```
Comment (timja) [inline vars/sectionBuildAndTest.groovy:104]: what if the docker ignore file already exists?

Comment (luigibk) [reply vars/sectionBuildAndTest.groovy:104]: Where? I looked for one.

Comment (timja) [reply vars/sectionBuildAndTest.groovy]: we never got a reliable fix, we turned off the files that were causing the issue in some builds which helped but sometimes people still hit it.

if this doesn't work I would either stop removing the file after use, or only remove it after the parallel stages have finished

**Comment by luigibk** [comment]
> worth a try

Why is that file deleted anyway?

**Comment by timja** [comment]
> > worth a try
> 
> Why is that file deleted anyway?

just because it was a file being created in the workspace, could just not delete it

**Pull request merged**